### DISCUSSION TITLE
old version

### DISCUSSION THREAD

**Discussion by BouarfaMahi** [mckaywrigley/chatbot-ui#1482] [Help]
Hey, I want to use the old version of the chatbot UI for my project. Is the new version built on top of the old one, or is it significantly different?

**Comment by secondtruth** [comment]
The new version is significantly different. The old version is available in the `legacy` branch and also as [container image](https://github.com/mckaywrigley/chatbot-ui/pkgs/container/chatbot-ui). But I would suggest to use one of the various forks like [smart-chatbot-ui](https://github.com/dotneet/smart-chatbot-ui).

**Reply by BouarfaMahi** [reply]
i will check, thanks
